src/main/java/com/google/opensesame/projects/ProjectEntity.java

```diff
@@ -73,6 +73,15 @@ public ProjectEntity(
     this.repositoryId = repositoryId;
     this.mentorIds = mentorIds;
     this.interestedUserIds = interestedUserIds;
+
+    // Empty lists are represented as null in datastore, so create empty lists if null:
+    // https://cloud.google.com/appengine/docs/standard/java/datastore/entity-property-reference#Using_an_empty_list
```

Comment:
Interesting, I will add this to the mentors page.

Reply:
Yeah this took quite a bit of time to debug haha. Happy to help.